src/cli/msig/staking.ts

```diff
@@ -48,8 +49,14 @@ export async function createStakingTmyAccount(options: StandardProposalOptions)
     const ownerAuthorityInfra = Authority.fromAccountPermission(ownerPermission);
     const activeAuthorityInfra = Authority.fromAccountPermission(activePermission);
 
+    // Preserve existing eosio.code permission for vesting.tmy
+    ownerAuthorityInfra.addCodePermission('vesting.tmy');
+    activeAuthorityInfra.addCodePermission('vesting.tmy');
```

Comment:
so the `staking.tmy` also needs to have the `vesting.tmy@eosio.code` permission? this does not seem necessary for this PR as staking nor vesting functionality has not changed. please explain